zetaclient/chains/bitcoin/observer/observer.go

```diff
@@ -694,18 +692,13 @@ func (ob *Observer) LoadLastBlock() error {
 		// #nosec G701 always in range
 		lastBN := int64(lastBlockNum.Num)
 		ob.SetLastBlockHeightScanned(lastBN)
-
-		//If persisted block number is too low, use the latest height
-		if (bn - lastBN) > maxHeightDiff {
```

Comment:
By default, scanning external chain from latest height makes a bit more sense than scanning from genesis because the integration of external chain usually(probably all) happened way later than their genesis. It's about just providing a default value, I think both are not that correct.